style.css

```diff
@@ -24,6 +24,60 @@ nav {
   position: relative;
 }
 
+.popup-card {
+  display: none;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  overflow: auto;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  background-color: rgba(0, 0, 0, 0.9);
+}
+
+.modal-body {
+  background-color: #fefefe;
+  margin: 5% auto;
+  padding: 1.25rem;
+  border: 1px solid #888;
+  width: 80%;
+  border-radius: 8px;
+}
+
+.technologies-list {
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+  margin: 0;
+  gap: 9px;
+}
+
+.technologies-list li {
+  padding: 8px 12px;
+  background: #ebecf0;
+  border-radius: 4px;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #344563;
+}
+
+.card-p {
+  margin-top: 2.25rem;
+}
+
+.modal-image {
```

Comment:
- [ ] The image is taking too much height on the mobile version. Kindly adjust the size of the image to match the design. If you are having issues adjusting the image size, I think you can use different images for mobile and desktop versions. 🥸️ Please also minimize the dark area to the sides of the popup and use the correct color. 🙂️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/NewtonOmbese/Newton-Portfolio/assets/17243325/5815b7d8-b6cb-45d5-8587-93858a4eee7a" width="300">|<img src="https://github.com/NewtonOmbese/Newton-Portfolio/assets/17243325/8f580c19-fcaa-4fdd-a8c3-ee6ed3707cc4" width="300">|